CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
 - **[FIX]** Handling of the colors on the `<PinIcon>`
+- **[UPDATE]** Change name of the `container` prop on the `<Text>` component into `tag` (harmonization)
```

Comment:
"breaking change" would be more appropriate

Reply:
Agreed